core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -108,7 +114,10 @@ public T failureResponseLogLevel(LogLevel failedResponseLogLevel) {
 
     /**
      * Returns the {@link LogLevel} to use when logging failure responses (e.g., failed with an exception).
+     *
+     * @deprecated It will be private in the future.
```

Comment:
private -> removed (ditto for the two other getters)

Reply:
OK. Changed it. But YOU ALREADY FIXED. 😄